### PR TITLE
Changes to FV3 for RRTMGP in ccpp-physics

### PR DESCRIPTION
**Pull request by dustinswales**
## Description
The PR contains some small changes to FV3 to accommodate recent improvements to RRTMGP in ccpp-physics.

The changes to RRTMGP are detailed here: https://github.com/NCAR/ccpp-physics/pull/733

### Issue(s) addressed
No github issues were opened.

## Testing
These changes were tested on Hera using the Intel compilers.
Not all of the errors addressed here were uncovered by the standard RTs, but rather showed up when running in the EMC workflow. I was able to replicate these errors by extending the RTs and running the tests for longer forecast times. 
Updates to the RT baselines are needed.

## Dependencies
https://github.com/NCAR/ccpp-physics/pull/733

### CODE REVIEW

**Review by climbfuji — CHANGES_REQUESTED** [review]
Submodule pointer for GFDL_atmos_cubed_sphere is not correct.

**Comment by climbfuji** [comment]
Submodule pointer for GFDL_atmos_cubed_sphere still seems to be off.

**Review by DusanJovic-NOAA — APPROVED** [review]

**Review by junwang-noaa — APPROVED** [review]

File: .gitmodules [hunk 1]
```diff
@@ -9,4 +9,4 @@
 [submodule "ccpp/physics"]
   path = ccpp/physics
   url = https://github.com/NCAR/ccpp-physics
-  branch = main
+  branch = main
```
Comment (climbfuji) [inline .gitmodules]: Let's fix the missing newline please @dustinswales

**Review by climbfuji — APPROVED** [review]

**Comment by climbfuji** [comment]
@junwang-noaa @DusanJovic-NOAA ready to merge

**Pull request merged**